res/controllers/Traktor-Kontrol-S4-MK3.js

```diff
@@ -2885,7 +2885,7 @@ class S4MK3 {
 
         const that = this;
         /* eslint no-unused-vars: "off" */
-        const meterConnection = engine.makeConnection("[Master]", "guiTick50ms", function(_value) {
+        const meterConnection = engine.makeConnection("[App]", "gui_tick_50ms_period_s", function(_value) {
```

Comment:
we should probably track this in an issue on the QML project board, otherwise this will silently break under QML. putting this on an `engine.beginTimer` instead should be sufficient.